qcengine/procedures/geometric.py

```diff
@@ -27,6 +27,9 @@ def compute(self, input_data: 'OptimizationInput', config: 'JobConfig') -> 'Opti
 
         geometric_input = input_data.dict()
 
+        # Temporary patch for geomeTRIC
+        geometric_input["initial_molecule"]["symbols"] = list(geometric_input["initial_molecule"]["symbols"])
+
         # Older QCElemental compat, can be removed in v0.6
```

Comment:
Since this PR depends on QCElemental 0.6, this may be a good time to remove this compat code.

Reply:
This compatibility code is required for backwards compatibility with `geometric`.